packages/web/e2e/uploadCollection.test.ts

```diff
@@ -9,15 +9,17 @@ import {
 import { test, waitForUser } from './test'
 import { openCleanBrowser } from './utils'
 
+// TODO: Enable track2 again, it was causing failures
```

Comment:
linear item w/ failure deets? seems fairly important to this test that multitrack uploads work

Reply:
Good call! added some items here: https://github.com/AudiusProject/audius-protocol/pull/8407/commits/524c109da88b184a8a503375af0f85f4041d0903